browserTest/test.js

```diff
@@ -0,0 +1,20 @@
+let webdriver = require('selenium-webdriver');
```

Comment:
Did `import` statements not work?

Reply:
Not supported by Node.js, one of the few things not supported by Node.js in ES6.